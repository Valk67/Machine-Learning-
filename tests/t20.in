.........o....***........
.....***oo.********......
..***ooxxx***.~~*****....
xoooxxxxxx***.*****......
.....xxxx***........xxo**
..................****oox
..**....~~~......****.~~.
...~~~~.~**....**.ooo..o.
.....~o~*.........**o..*.
...oo...~~~~.....******..
