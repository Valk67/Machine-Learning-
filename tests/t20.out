S........o....***........
v....***oo.********......
>v***ooxxx***.~~*****....
xvooxxxxxx***.*****......
.v...xxxx***........xxo**
.>>>>>>>>>>v......****oox
..**....~~~v.....****.~~.
...~~~~.~**v...**.ooo..o.
.....~o~*..>>>>>>>>>>>v*.
...oo...~~~~.....*****>>G
